apps/example/plugins/withDesugaring.js

```diff
@@ -0,0 +1,24 @@
+// https://github.com/expo/expo/discussions/31071#discussioncomment-11128404
+const { withAppBuildGradle } = require('expo/config-plugins')
+
+module.exports = function withCustomAppBuildGradle(config) {
```

Comment:
nit: Could we add a quick comment about what this plugin does and why it's needed?